### PR TITLE
Issue #476 Fix: Blank screen is getting displayed for a second on opening a Program from my Programs tab.

### PR DESCRIPTION
**Pull request by niteshkesarkar**

### CODE REVIEW

File: src/app/client/src/app/modules/program/components/program-nominations/program-nominations.component.ts [hunk 1]
```diff
@@ -220,6 +221,7 @@ export class ProgramNominationsComponent implements OnInit, AfterViewInit, OnDes
       this.showNominationsComponent = true;
     }, error => {
       this.tosterService.error('User onboarding failed');
```
Comment (pallakartheekreddy) [inline src/app/client/src/app/modules/program/components/program-nominations/program-nominations.component.ts:223]: only in `else` condition, you added

File: src/app/client/src/app/modules/program/components/program-nominations/program-nominations.component.ts [hunk 2]
```diff
@@ -353,6 +357,7 @@ export class ProgramNominationsComponent implements OnInit, AfterViewInit, OnDes
       // TODO: navigate to program list page
       const errorMes = typeof _.get(error, 'error.params.errmsg') === 'string' && _.get(error, 'error.params.errmsg');
       this.toasterService.error(errorMes || this.resourceService.messages.emsg.project.m0001);
```
Comment (pallakartheekreddy) [inline src/app/client/src/app/modules/program/components/program-nominations/program-nominations.component.ts:359]: check to hear as well

File: src/app/client/src/app/modules/program/components/program-nominations/program-nominations.component.ts [hunk 3]
```diff
@@ -410,6 +415,7 @@ export class ProgramNominationsComponent implements OnInit, AfterViewInit, OnDes
...
```
Comment (pallakartheekreddy) [inline src/app/client/src/app/modules/program/components/program-nominations/program-nominations.component.ts:417]: check to hear as well

File: src/app/client/src/app/modules/program/components/program-nominations/program-nominations.component.ts (hunk 1 above)
Comment (niteshkesarkar) [reply src/app/client/src/app/modules/program/components/program-nominations/program-nominations.component.ts:223]: there are 4 api call in `ngOnInit()`  those are `
    this.getNominationList();
    this.getProgramDetails();
    this.getNominationCounts();
    this.getProgramCollection();`. So currently I kept `public showLoader = true;` bu default. and if every api call get successfully executed then added `this.showLoader = false;` also added this line in every api's error case to hide the loader after failure. 

If we want to have other logic then we can initialise `loader = true` before every api call and add `loader = false` in success and error case of every api. 

@pallakartheekreddy - Is it ok to have first approach or should I use second one? Is there any other better solution to this then please let me know?

**Pull request closed without merging**